frontend/components/Comment.js

```diff
@@ -5,7 +5,6 @@
 import { Component } from 'react'
 
 import Tippy from 'tippy.js'
-import $ from 'jquery'
```

Comment:
I wonder if this was added because it's a dep of `tippy.js`, even if unused in the file.